src/Guest.js

```diff
@@ -0,0 +1,26 @@
+class Guest {
+  constructor(id, name) {
+    this.id = id;
+    this.name = name;
+    this.guestBookings = null;
+  }
+
+  getGuestFirstName() {
+    const firstName = this.name.split(' ')[0];
+    return firstName;
+  }
+
+  getGuestBookings(hotel) {
+    hotel.addRoomInfoToBookings()
+    this.guestBookings = hotel.findGuestBookings(this.id)
```

Comment:
I would try and get functionality for each class to do SPR, so no overlap or sharing of methods.

Reply:
That's a good catch. Thanks! I might try to make this a parent/child relationship, or just shift all the guest booking related functionality to the Guest class.